packages/coverage-istanbul/src/provider.ts

```diff
@@ -199,7 +199,7 @@ export class IstanbulCoverageProvider extends BaseCoverageProvider<ResolvedCover
       }
 
       // Make sure file is not served from cache so that instrumenter loads up requested file coverage
-      await transform(`${filename}?v=${cacheKey}`)
+      await transform(`${filename}?t=${cacheKey}`)
```

Comment:
Looks like this doesn't work. Vite does not load the module again.

Reply:
We relied on Vite removing the query parameter before. Now done manually when checking for includes.